cfme/scripting/quickstart.py

```diff
@@ -0,0 +1,256 @@
+from __future__ import print_function
+
+import os
+import sys
+import argparse
+import subprocess
+import json
+import hashlib
+
+try:
+    import distro
+except ImportError:
+    sys.path.insert(0, os.path.join(
+        os.path.dirname(os.path.dirname(
+            os.path.dirname(__file__))),
+        'requirements', 'quickstart',
+        'distro-1.0.4-py2.py3-none-any.whl'))
+    import distro
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--mk_virtualenv", default="../cfme_venv")
```

Comment:
**REQUIRED:** Something I only just noticed on running again, can we have consistency between ```_``` and ```-```